src/Auditor.jl

```diff
@@ -120,6 +135,7 @@ function audit(prefix::Prefix, src_name::AbstractString = "";
                     return 0
                 catch e
                     if $(repr(verbose))
+                        println("##vso[task.logissue type=warning;;]Could not dlopen $(repr(f))")
```

Comment:
Is this supposed to be here?

Reply:
It was intended by me to be there, yes, but it's open for discussion.  I can never find this warning :grimacing:

Reply:
The reason I ask is because this will print a weird message on people's computers who aren't doing anything with Azure, right?

Reply:
Yes, that's true :disappointed: